src/js/components/sharedComponents/filterSidebar/FilterSidebar.jsx

```diff
@@ -17,7 +17,7 @@ const defaultProps = {
 };
 
 const propTypes = {
-    options: PropTypes.arrayOf(PropTypes.string),
+    options: PropTypes.arrayOf(PropTypes.shape({})),
```

Comment:
Can we have what this shape is going to be or is it going to change or is dynamic?
`PropTypes.shape({ title: PropTypes.string, className: PropTypes.string, tooltip: PropTypes.element })`

Reply:
As you can see in line 31 below, those properties aren't always going to be in that object so it doesn't make sense to me to define them in our prop types. Unless there's a way to say certain object properties are requried while others are not.

Reply:
`.isRequired ` reference https://reactjs.org/docs/typechecking-with-proptypes.html.

Isn't title always going to be there?